src/promptflow-tools/promptflow/tools/aoai.py

```diff
@@ -6,12 +6,13 @@
 from promptflow.contracts.types import PromptTemplate
 # Avoid circular dependencies: Use import 'from promptflow._internal' instead of 'from promptflow'
 # since the code here is in promptflow namespace as well
-from promptflow._internal import enable_cache, ToolProvider, tool, register_apis
+from promptflow._internal import enable_cache, ToolProvider, tool, register_apis, record_decorator
```

Comment:
should add tool changes after new pf pypi package is released.

Reply:
Will remove this change in next patch